lib/beaker/dsl/install_utils/puppet_utils.rb

```diff
@@ -583,25 +588,22 @@ def install_puppetlabs_dev_repo ( host, package_name, build_version,
                           repo_filename,
                           platform_configs_dir)
 
-            link = "%s/%s/%s/repos/%s/%s%s/PC1/%s/" %
-              [ dev_builds_url, package_name, build_version, variant,
-                fedora_prefix, version, arch ]
-
-            if not link_exists?( link )
-              link = "%s/%s/%s/repos/%s/%s%s/products/%s/" %
-                [ dev_builds_url, package_name, build_version, variant,
-                  fedora_prefix, version, arch ]
-            end
-
-            if not link_exists?( link )
-              link = "%s/%s/%s/repos/%s/%s%s/devel/%s/" %
+            link = nil
+            package_repos = options[:dev_builds_repos].nil? ? [] : [options[:dev_builds_repos]]
+            package_repos.push(['PC1', 'products', 'devel']).flatten!
```

Comment:
I don't think PC1 should be present in beaker, even as a fallback. The reason is that I can imagine a scenario where a caller of this method, e.g. puppetserver, wants to use puppet-agent from PC2, but forgets to specify `dev_builds_repos`, and pulls in the version of PC1, and things just might "work" and fail sometime later due to versioning issues between puppet-agent and puppetserver. I think it would be better for the caller to always specify the PCN repos, and maybe fall back to "products and devel"... /cc @haus @jeffmccune